tools/install.sh

```diff
@@ -155,10 +155,15 @@ chmod 770 ../smarty/templates_c
 
 # Changing group to 'www-data' or 'apache' to give permission to create directories in Document Repository module
 # Detecting distribution
-os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')
+if ! os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')  2> /dev/null
```

Comment:
```suggestion
if ! os_distro=$(hostnamectl 2>/dev/null)
```

Given that my solution doesn't work, please change the line to this instead. It's a bit shorter and properly suppresses the error message (`2>/dev/null` should be inside the brackets).